bin/cron/send_permission_email_reminders

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+# This script will send reminder emails for parent permission requests that
+# haven't been granted yet. Intended for Colorado Privacy Act compliance.
+
+require 'active_support/core_ext/numeric/time'
+require 'date'
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = <<~BANNER
+    Usage: #{File.basename(__FILE__)} [options]
+
+    This script will send reminder emails to parents who haven't yet granted
+    permission for their child's code.org account.
+
+    Options:
+  BANNER
+
+  opts.on('--max-age', 'Timestamp of the oldest requests to send reminders for. Defaults to 7 days ago.') do |max_reminder_age|
```

Comment:
tiny usability feature request, totally not needed: I like a `--max-age-days` option to be able to quickly say 5 days rather than 7, as implemented, I have to lookup the timestamp for 5 days ago and then plug it in